### DISCUSSION TITLE
Does pdm use central installation caches with virtual environments?

### DISCUSSION THREAD

**Discussion by kingyue737** [pdm-project/pdm#2147] [Q&A] [answered]
I'm looking for a python package manager similar to `pnpm` in order to save disk storage. The documentation shows that the original `__pypackages__` is not recommended anymore. I wonder whether `pdm` still supports central installation caches in virtual environments.

**Comment by frostming** [comment, marked as answer]
Yes

**Reply by jordantshaw** [reply]
Can you elaborate?

**Reply by athrunsun** [reply]
@jordantshaw It is in the documentation: https://pdm-project.org/latest/usage/config/#central-installation-caches
